test/express.js

```diff
@@ -44,7 +44,7 @@ describe('presets', function() {
 
           expect(chunk).to.include('### Dependencies');
           expect(chunk).to.include('type-is@~1.6.3');
-          expect(chunk).to.include(' - deps: mime-types@~2.1.1\n');
+          expect(chunk).to.include('- deps: mime-types@~2.1.1\n');
```

Comment:
This should be fixed now. Can you please remove this change? I'll merge this as soon as I get a Mac. Thanks.